Sources/PackageDescription/JSON.swift

```diff
@@ -0,0 +1,52 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+/// A very minimal JSON type to serialize the manifest.
+// FIXME: A more complete JSON type is already present in Basic but
+// it needs to discussed if and how PackageDescription should handle
+// dependencies.
+enum JSON {
+    case null
+    case int(Int)
+    case double(Double)
+    case bool(Bool)
+    case string(String)
+    case array([JSON])
+    case dictionary([String: JSON])
+}
+
+extension JSON {
+    /// Converts the JSON to string representation.
+    // FIXME: No escaping implemented for now.
+    func toString() -> String {
+        switch self {
+        case .null:
+            return "null"
+        case .bool(let value):
+            return value ? "true" : "false"
+        case .int(let value):
+            return value.description
+        case .double(let value):
+            return value.debugDescription
+        case .string(let value):
+            return "\"" + value + "\""
```

Comment:
Yes, `Basic.JSON` already has all this in place which should be used here somehow but that needs to be thought and discussed so this is a very light weight JSON type right now. see https://github.com/apple/swift-package-manager/pull/604/files/36ea1c997178206172ff5df8ff1782a68703baff#diff-3e9fa06405ae4eaad8f737ebdfaefefbR12